src/persistence/graph6.jl

```diff
@@ -0,0 +1,125 @@
+
+function _bv2int(x::BitVector)
+  assert(length(x) <= 8 * sizeof(Int))
+  acc = 0
+  for i = 1:length(x)
+    acc = acc << 1 + x[i]
+  end
+  return acc
+end
+
+function _int2bv(n::Int, k::Int)
+  bitstr = lstrip(bits(n), '0')
+  l = length(bitstr)
+  padding = k - l
+  bv = falses(k)
+  for i = 1:l
+    bv[padding+i] = (bitstr[i] == '1')
+  end
+  return bv
+end
+
+function R(_x::BitVector)::Vector{UInt8}
```

Comment:
Is this function name from the g6 reference implementation? If not, why the short name? Same goes for the others. Should we call them `g6_R`, `g6_Rp`, etc.?